packages/cli/src/commands/bisect/index.ts

```diff
@@ -148,14 +143,13 @@ export default async function main(client: Client): Promise<number> {
   }
 
   output.spinner('Retrieving deployments…');
-  const [badDeployment, goodDeployment] = await Promise.all([
-    badDeploymentPromise,
-    goodDeploymentPromise,
-  ]);
+
+  // `getDeployment` cannot be parallelized because it might prompt for login
```

Comment:
This is the main bug fix. When we parallelize the lookups for the good and bad deployment, if the current user's auth doesn't have the proper scope, the user will be prompted to log in with SAML. This is async, though, which stuffs that prompt into your face when you are trying to interact with other prompts, like selection a subpath.